query-service-impl/src/test/resources/prometheus_request_handler.conf

```diff
@@ -0,0 +1,42 @@
+{
+  name = raw-service-prometheus-handler
+  type = prometheus
+  clientConfig = ""
+  requestHandlerInfo {
+    tenantAttributeName = tenant_id
+    prometheusViewDefinition {
+      viewName = rawServiceView
+      metricMap {
```

Comment:
each metric map entry, can we define something like
```
attribute: API.numCalls
            metric : { metricName: "num_calls",
              metricType: "GAUGE" }
```

Reply:
Can you log a ticket for this so we can change this structure to more readable and align with `SERVICE.numCall` attribute pattern?